.github/workflows/create-release.yaml

```
@@ -1,369 +1,153 @@
-            echo "4. ✅ **PR Created**: [#${{ steps.create_pr.outputs.pr_number }}](${{ steps.create_pr.outputs.pr_url }}) - [View Changes](${{ steps.create_pr.outputs.pr_url }}/files)" >> $GITHUB_STEP_SUMMARY
-          fi
-
-          echo "" >> $GITHUB_STEP_SUMMARY
-          echo "---" >> $GITHUB_STEP_SUMMARY
-          echo "" >> $GITHUB_STEP_SUMMARY
-          echo "### Workflow Inputs" >> $GITHUB_STEP_SUMMARY
-          echo "- **Current Tag**: \`${{ github.event.inputs.tag_name }}\`" >> $GITHUB_STEP_SUMMARY
-          echo "- **Next Tag**: \`${{ github.event.inputs.next_odh_tag }}\`" >> $GITHUB_STEP_SUMMARY
-          echo "- **Dry Run**: \`${{ github.event.inputs.dry_run }}\`" >> $GITHUB_STEP_SUMMARY
-          echo "- **Base Branch**: \`${{ github.ref_name }}\`" >> $GITHUB_STEP_SUMMARY
+name: Create Tag and Release with Changelog
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)